tests/lib/rules/no-sequences.js

```diff
@@ -47,17 +47,23 @@ ruleTester.run("no-sequences", rule, {
         "if ((doSomething(), !!test));",
         "switch ((doSomething(), !!test)) {}",
         "while ((doSomething(), !!test));",
-        "with ((doSomething(), val)) {}"
+        "with ((doSomething(), val)) {}",
+        { code: "(0,eval)(\"foo()\");", options: [{ ignoreParenthesized: false }] },
+        // Below is a valid case since ignoreParenthesized simply ignores anything in parens
```

Comment:
Nope, I meant below since although we set `allowIndirectEval` to `false` it passes because of `ignoreParenthesized` being `true` by default. That said looks like the comment is not clear enough. Any suggestions to make it better?